dev/tasks/conda-recipes/arrow-cpp/meta.yaml

```diff
@@ -81,6 +82,7 @@ outputs:
         - lz4-c
         - numpy
         - orc  # [unix]
+        - openssl
```

Comment:
This is not already a dependency of `aws-sdk-cpp`, `grpc-cpp` or `google-cloud-cpp`?

Reply:
Yes, but Gandiva also links directly to it and thus it should also be listed directly here to also get rebuilds for OpenSSL version updates.

Reply:
Ok, thanks for the explanation.